util/verifyTarget.sh

```diff
@@ -184,7 +183,8 @@ for PROJECT in $PROJECTS; do echo "Process $PROJECT ..."
   fi
 
   echo ""
-  if [[ ${includeSources} ]]; then
+  if [[ ! ${includeSources} ]]; then
```

Comment:
I don't get it

Reply:
it's linked to the includeSources declaration above that i removed.
before, the includeSources was always present and the check was to inform if the mirror include or not the sources.
now, the logging is correct

Reply:
if nothing is set, the log is including source and include sources is set
to exclude, the -x param is used and was incorrectly reporting including sources in log